eras/shelley/impl/src/Cardano/Ledger/Shelley/Orphans.hs

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS_GHC -Wno-dodgy-exports #-}
+
+-- This "dodgy" option is needed because Cardano.Ledger.Orphans exports nothing
+-- but makes a bunch of orphan instances, and it is these orphan instances we
+-- want to be exported. The dodgy-exports Flag does not seem to understand about instances.
+
+module Cardano.Ledger.Shelley.Orphans
+  {-# DEPRECATED "Use 'import Cardano.Ledger.Orphans' instead" #-}
+  ( module X,
+  )
+where
+
+import Cardano.Ledger.Orphans as X ()
```

Comment:
GHC option is not needed, it is indeed warning you about unnecessary thing. Instances are reexported implicitly. So this is a correct approach:
```haskell
module Cardano.Ledger.Shelley.Orphans
  {-# DEPRECATED "Use 'import Cardano.Ledger.Orphans' instead" #-}
  ( )
where

import Cardano.Ledger.Orphans ()
```

Reply:
So if you do this, it gives another error, something like Cardano.Ledger.Orphans does not import anything, and is redundant

Reply:
If you do without `as X` it will not give this error. So, just doing this:
```
import Cardano.Ledger.Orphans ()
```
should not give you any errors

Reply:
So I fixed this, as suggested, since it worked.